content/en/blog/_posts/2021-04-08-PodSecurityPolicy-Past-Present-and-Future.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: blog
+title: "PodSecurityPolicy: Past, Present, and Future"
```

Comment:
This is a wonderful read! Tim covered the review, and I had just one minor addition from years of SEO work. 

This article will show up more often in Google/Bing/DDG searches and help more people if it's title and slug align to a searcher's intent. My assumption there is people will search `PSP deprecation` or PodSecurityPolicy deprecation`, so it'd be great to get those terms in both places. If you want to keep a similar format: `the How, What, and When of PSP Deprecation` could be fun. Or something playful like `Don't Panic: Updates on PodSecurityPolicy Deprecation`. Either way, the keyword in the title (and slug) helps a lot with ranking.

FWIW as well, I found this article is in the top position on search and it's got some great additional context. Adding here for others to see: https://www.antitree.com/2020/11/pod-security-policies-are-being-deprecated-in-kubernetes/

Reply:
Thank you for the reminder about that blog post! It has some misinformation in it, so I sent a note to Mark to correct it. 😊